Makefile

```diff
@@ -49,6 +49,7 @@ tools: $(GOLANGCI_LINT) $(MISSPELL) $(GOCOVMERGE) $(STRINGER) $(GOJQ) $(FIELDALI
 
 PROTOBUF_VERSION = 3.19.0
 PROTOBUF_OS = linux
+UNAME_S := $(shell uname -s)
```

Comment:
I noticed protoc wasn't working on my mac as this variable wasn't actually set to anything, I believe it was removed with some cleanup at some point. It could be nice to run the entire CI on multiple machines to ensure it continues to build on them, though the notable step is this one I think

https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/main/.github/workflows/ci.yml#L41